packages/terra-overlay/src/LoadingOverlay.jsx

```diff
@@ -8,13 +8,13 @@ import styles from './Overlay.module.scss';
 
 const cx = classNames.bind(styles);
 
-const { BackgroundStyles, zIndexes } = Overlay.Opts;
+const { BackgroundStyles } = Overlay.Opts;
 
 const propTypes = {
   /**
    * The visual theme to be applied to the overlay background. Accepts 'light', 'dark', and 'clear' or BackgroundStyles.LIGHT, BackgroundStyles.DARK, and BackgroundStyles.CLEAR.
    */
-  backgroundStyle: PropTypes.oneOf(['light', 'dark', 'clear', BackgroundStyles]),
+  backgroundStyle: PropTypes.oneOf(['light', 'dark', 'clear']),
```

Comment:
Update the prop description as well, since the `BackgroundStyles` enum has been removed from the proptype.

Reply:
done here https://github.com/cerner/terra-core/pull/2878/commits/17082193960c6ecc5da77a7adfd102efc4a95d1c.